app/assets/javascripts/discourse/app/components/user-status-picker.gjs

```diff
@@ -49,6 +49,8 @@ export default class UserStatusPicker extends Component {
           @icon={{@status.emoji}}
           @didSelectEmoji={{this.emojiSelected}}
           @btnClass="btn-emoji"
+          @modalForMobile={{false}}
```

Comment:
We can't have a modal for the picker here, because the status picker is already a modal and we (rightfully) don't support double modal.